src/connection.c

```diff
@@ -168,9 +175,28 @@ int createNopollConnection(noPollCtx *ctx)
 				ParodusError("Error connecting to server\n");
 				ParodusError("RDK-10037 - WebPA Connection Lost\n");
 				// Copy the server address from config to avoid retrying to the same failing talaria redirected node
-				allow_insecure = parse_webpa_url (get_parodus_cfg()->webpa_url,
+				if (get_parodus_cfg()->acquire_jwt == 0)
```

Comment:
This code seems to be repeated several times.
Wouldn't it be more readable if we made a function to encapsulate it ?

Reply:
It would/does.  I don't know that we should hold this change up for that vs. add it as an improvement.

Reply:
OK, I'm fine with that.
I've got another change that needs to go in anyway.
But, can you please comment on my other comment, below.
If I've understood it right, doesn't this new code loop forever, trying to connect to the
jwt url, if there is one ?
What we want is for it to retry the jwt url N times, then revert to the default url.